ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2022.3.34f1
```

Comment:
Kan je dit reverten? Het team zit op editor 2022.3.34f1, en we hebben afgesproken om gepland als team te upgraden om te voorkomen dat we opeens met z'n allen editors ad hoc moeten updaten

Reply:
done